herd/AArch64Sem.ml

```diff
@@ -107,6 +111,7 @@ module Make
       module Mixed = M.Mixed(SZ)
 
       let (>>=) = M.(>>=)
+      let (let*) = M.(>>=)
```

Comment:
Just a question for future reference: do we now prefer let-binders in new code? Would the idea be to define them globally upon use?

Reply:
I like them more but I don't think we have to convert existing code, not sure if @maranget has a strong opinion on this.

Reply:
Hi @relokin, I have no strong opinion on the subject. Generally, I am not found of constraints in coding beyond decent indentation and variable names....